test/snippets.cpp

```diff
@@ -80,12 +80,10 @@ usingStrings()
         json::string jstr1 = "helloworld";
         json::string jstr2 = "world";
```

Comment:
This would be a better example if the two strings did not share the word "world"